### PR TITLE
documentation: Fix link, typos, order

### PR DESCRIPTION
**Pull request by speller26**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#commit-your-change)
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-braket-sdk-python/blob/main/README.md) and [API docs](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by wrasmuss — DISMISSED** [review]
Nice. Is chemestry having a disorganized lab? I bet we'll have better luck with chemistry.

File: doc/examples-getting-started.rst [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ##############################
-Simple circuits and algorithms
+Getting started
 ##############################
 
-Learn more about working with advanced circuits and algoritms.
+Get started with Amazon Braket with some introductory examples.
```
Comment (wrasmuss) [inline doc/examples-getting-started.rst:5]: Maybe Get started with Amazon Braket: introductory examples. The double with in the sentence is a little weird.

File: doc/examples-getting-started.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ##############################
-Simple circuits and algorithms
+Getting started
```
Comment (wrasmuss) [inline doc/examples-getting-started.rst:2]: These are already sentence case. So that's probably right, but are you sure we don't want title case?

**Review by wrasmuss — CHANGES_REQUESTED** [review]

File: doc/examples-getting-started.rst (hunk 2 above)
Comment (speller26) [reply doc/examples-getting-started.rst:2]: I was considering it, but I think that's a story for another day, when we sweep over every example

**Review by wrasmuss — APPROVED** [review]
Looks good.

**Pull request merged**